tests/testthat/helper_skip_if_fast.R

```diff
@@ -0,0 +1,8 @@
+skip_if_fast <- function() {
```

Comment:
yeah, there is some tension between adhering to our code standard and adhering to the code standard of testthat, which uses snake_case instead of camelCase.